wagtailcaptcha/models.py

```diff
@@ -18,14 +18,10 @@ def __init__(self, *args, **kwargs):
         self.form_builder = WagtailCaptchaFormBuilder
 
     def process_form_submission(self, form):
-        super(AbstractEmailForm, self).process_form_submission(form)
-
-        if self.to_address:
-            content = ''
-            for x in form.fields.items():
-                if not isinstance(x[1], ReCaptchaField):  # exclude ReCaptchaField from notification
-                    content += '\n' + x[1].label + ': ' + text_type(form.data.get(x[0]))
-            send_mail(self.subject, content, [self.to_address], self.from_address,)
+        if WagtailCaptchaFormBuilder.CAPTCHA_FIELD_NAME in form.fields.keys():
+            form.fields.pop(WagtailCaptchaFormBuilder.CAPTCHA_FIELD_NAME)
+
+        return super(WagtailCaptchaEmailForm, self).process_form_submission(form)
```

Comment:
Looking at the django-recaptcha code, [looks like it will still propagate the captcha challenge](https://github.com/praekelt/django-recaptcha/blob/develop/captcha/fields.py#L58). It doesn't matter to much if that data is in the db, but if we want it not stored we can always instead just do:
`
     def process_form_submission(self, form):
        if WagtailCaptchaFormBuilder.CAPTCHA_FIELD_NAME in form.cleaned_data.keys():
             form.cleaned_data.pop(WagtailCaptchaFormBuilder.CAPTCHA_FIELD_NAME)
 
         return super(WagtailCaptchaEmailForm, self).process_form_submission(form)`

At least that way the code isn't entirely duplicated. Although now I'm curious why [AbstractEmailForm isn't using cleaned_data](https://github.com/torchbox/wagtail/blob/48296e756858850042ffaabb0bf023cf6f144c49/wagtail/wagtailforms/models.py#L295)...